install/helm/openchoreo-observability-plane/README.md

```diff
@@ -0,0 +1,183 @@
+# OpenChoreo Observability Plane
```

Comment:
Do we need the readme here? I feel like this belongs to the documentation. WDYT?